# GameStatus.msg
## Game status : Robot type, hp, mode

time stamp

uint8 robot_type

uint16 red_std_hp
uint16 red_hro_hp
uint16 red_sty_hp
uint16 blu_std_hp
uint16 blu_hro_hp
uint16 blu_sty_hp

uint8 mode
